### DISCUSSION TITLE
[eslint-config] Set prettier as optional or remove it

### DISCUSSION THREAD

**Discussion by smujaddid** [adonisjs/core#4732] [Ideas]
I suggest to **remove** `prettier` from `eslint-config` or set it as **optional**.
By making it optional, if the user installed `prettier` as a dependency then the `eslint-prettier-plugin` will be loaded and used, otherwise don't load it at all.

Personally, I abandoned `prettier` usage as I prefer to use both [ESLint Stylistic](https://eslint.style) and [Stylelint](https://stylelint.io/) for formatting code and css files.
I know both eslint stylistic and stylelint are not the same as `prettier`, especially for HTML formatting. But, making it optional would be helpful for some developers who don't want to use `prettier` in any form.

I can always create my own config to use for my Adonis-based project, but if I can use the provided plugin by Adonis, then I prefer that instead.

This is just an opinion and a small request. I don't mean to disrespect anyone's preference. Everyone has their own setup and preference. That's it.

Thank you.

**Comment by moooza** [comment]
i disable prettier with this config in eslint.config.js at root directory
```javascript
import { configApp } from '@adonisjs/eslint-config'
export default configApp({
  rules: {
    'prettier/prettier': 'off',
  }
})
```
